wgpu-hal/src/dx12/shader_compilation.rs

```diff
@@ -88,21 +88,48 @@ mod dxc {
         pub library: hassle_rs::DxcLibrary,
         // Has to be held onto for the lifetime of the device otherwise shaders will fail to compile
         _dxc: hassle_rs::Dxc,
+        // Only used when dxil.dll has an explicit path to avoid automatic validation silently failing.
+        pub validator: Option<hassle_rs::DxcValidator>,
```

Comment:
```suggestion
        validator: Option<hassle_rs::DxcValidator>,
```
This doesn't need to be `pub`, right?

Reply:
None of those need to be pub, I forgot to change them when I refactored the DXC stuff into a self contained file.